### PR TITLE
CASMPET-5748 Fix for vshasta

### PR DESCRIPTION
**Pull request by cdelatte-hpe**
### Summary and Scope

Fixes the 1.3 ceph install for vshasta

- Fixes: [CASMPET-5748](https://jira-pro.its.hpecorp.net:8443/browse/CASMPET-5748)
- Requires:
- Relates to:

#### Issue Type

- Bugfix Pull Request

### Prerequisites

- [ ] I have included documentation in my PR (or it is not required)
- [x] I tested this on internal system (if yes, please include results or a description of the test)
- [x] I tested this on a vshasta system (if yes, please include results or a description of the test)

### CODE REVIEW

**Review by bklei — APPROVED** [review]
looks good after addressing mr. borman's comments

File: boxes/ncn-node-images/kubernetes/files/resources/metal/containerd/config.toml [hunk 1]
```diff
@@ -22,26 +22,26 @@ oom_score = -999
   [plugins."io.containerd.grpc.v1.cri".registry]
     [plugins."io.containerd.grpc.v1.cri".registry.mirrors]
       [plugins."io.containerd.grpc.v1.cri".registry.mirrors."dtr.dev.cray.com"]
-        endpoint = ["http://pit.nmn:5000/v2/dtr.amslabs.hpecorp.net", "http://pit.nmn:5000/v2/dtr.dev.cray.com", "http://pit.nmn:5000", "http://registry.local/v2/dtr.amslabs.hpecorp.net", "http://registry.local/v2/dtr.dev.cray.com", "http://registry.local"]
+        endpoint = ["http://pit.nmn:5000/v2/dtr.amslabs.hpecorp.net", "http://pit.nmn:5000/v2/dtr.dev.cray.com", "http://pit.nmn:5000", "https://registry.local/v2/dtr.amslabs.hpecorp.net", "https://registry.local/v2/dtr.dev.cray.com", "https://registry.local", "dummy://dtr.amslabs.hpecorp.net", "dummy://dtr.dev.cray.com"]
```
[the remark this answers is not in the document]
Comment (cdelatte-hpe) [reply boxes/ncn-node-images/kubernetes/files/resources/metal/containerd/config.toml]: fixed

File: boxes/ncn-node-images/kubernetes/files/resources/metal/containerd/config.toml [hunk 2]
```diff
@@ -22,26 +22,26 @@ oom_score = -999
...
       [plugins."io.containerd.grpc.v1.cri".registry.mirrors."arti.dev.cray.com"]
-        endpoint = ["http://pit.nmn:5000/v2/arti.amslabs.hpecorp.net", "http://pit.nmn:5000/v2/arti.dev.cray.com", "http://pit.nmn:5000", "http://registry.local/v2/arti.amslabs.hpecorp.net", "http://registry.local/v2/arti.dev.cray.com", "http://registry.local"]
+        endpoint = ["http://pit.nmn:5000/v2/arti.amslabs.hpecorp.net", "http://pit.nmn:5000/v2/arti.dev.cray.com", "http://pit.nmn:5000", "https://registry.local/v2/arti.amslabs.hpecorp.net", "https://registry.local/v2/arti.dev.cray.com", "https://registry.local", "dummy://arti.amslabs.hpecorp.net", "dummy://arti.dev.cray.com"]
```
[the remark this answers is not in the document]
Comment (cdelatte-hpe) [reply boxes/ncn-node-images/kubernetes/files/resources/metal/containerd/config.toml]: fixed

**Review by rustydb — APPROVED** [review]
I squashed the commits, in general that makes it easier to revert if we needed to (not that I think we will). I added the JIRA to the commit message to help history as well.

**Pull request merged**